src/gui/render/array_buffer/array_buffer.hpp

```diff
@@ -189,9 +199,8 @@ ArrayBuffer::ArrayBuffer(std::vector<T> buffer_data, GLuint devisor, buffer_type
 
 template <class T>
 void
-ArrayBuffer::update(std::vector<T> buffer_data, GLuint devisor) {
-
-    devisor_ = devisor;
+ArrayBuffer::update(std::vector<T> buffer_data, GLuint divisor) {
+    divisor_ = divisor;
 
     bool classes_are_equivelent = operator==(array_buffer_type::init(T()));
```

Comment:
what is this

how are you calling a two-parameter operator with a single argument?